### ISSUE TITLE
Code `pre` has bad css for lightmode

### ISSUE THREAD

**Issue by ikshyza** [friendofsvelte/tipex#4]
While using your editor, it has some issues like this one. I am mostly in light mode, so please fix.

![image](https://github.com/friendofsvelte/tipex/assets/120033434/5d564bf8-14c7-421b-8759-b57138d30178)

By the way, I did this to temporarily fix the issue:
```css
    .html-editor :global(.tipex-editor .tipex-content-section pre) {
        border-radius: 12px;
    }

    
    .html-editor :global(.tipex-editor .tipex-content-section pre code) {
        background-color: rgba(240, 248, 255, 0.144);
        color: rgb(255, 255, 255);
    }
```

**Comment by Bishwas-py** [comment]
This commit adds the fixes the issue: d950fa09badb9db221c20803cee134c3d8ad2332 . Checkout the new [version](https://www.npmjs.com/package/@friendofsvelte/tipex/v/0.0.1-meta-stable.2)
![image](https://github.com/friendofsvelte/tipex/assets/42182303/ca883e79-8cf1-4baf-a1da-7bfe4033bb58)

**Comment by ikshyza** [comment]
Feels pretty good, by the way how to disable the code highlighting?

**Comment by Bishwas-py** [comment]
Fixed in version `0.0.1-act.3`
